lib/manageiq/appliance_console/saml_authentication.rb

```diff
@@ -90,13 +90,21 @@ def fetch_idp_metadata
 
       def copy_apache_saml_configfiles
         debug_msg("Copying Apache SAML Config files ...")
-        copy_template(HTTPD_CONFIG_DIRECTORY, "manageiq-remote-user.conf")
+        # introduced manageiq-remote-user-saml.conf in 4.7.1.
+        # this File.exist? will no longer be needed after 4.7.2
+        if File.exist?(File.join(HTTPD_CONFIG_DIRECTORY, "manageiq-remote-user-saml.conf"))
+          copy_template(HTTPD_CONFIG_DIRECTORY, "manageiq-remote-user-saml.conf")
+        else
+          copy_template(HTTPD_CONFIG_DIRECTORY, "manageiq-remote-user.conf")
+        end
         copy_template(HTTPD_CONFIG_DIRECTORY, "manageiq-external-auth-saml.conf")
       end
 
       def remove_apache_saml_configfiles
         debug_msg("Removing Apache SAML Config files ...")
+        # legacy systems may still have it stored as the old name
         remove_file(HTTPD_CONFIG_DIRECTORY.join("manageiq-remote-user.conf"))
```

Comment:
Let's change this comment to be only on the line it comments on.  Otherwise it appears like it applies to the whole "paragraph" of code.

```suggestion
        remove_file(HTTPD_CONFIG_DIRECTORY.join("manageiq-remote-user.conf")) # legacy systems may still have it stored as the old name
```

Reply:
thnx.
think we're getting away from same line comments (rubocop)
Will change wording to make more clear

Reply:
I disagree with rubocop on that :)